components/discord_bot/actions/add-role/add-role.mjs

```diff
@@ -0,0 +1,57 @@
+import common from "../common.mjs";
```

Comment:
Looks like the only thing discord_bot/actions/common.mjs adds on top of discord_bot/common.mjs is the `channelId` prop. Since this action doesn't use `channelId`, I think we could just include discord_bot/common.mjs in this component and any others that don't use `channelId`.

`import common from "../..common.mjs`